be1-go/hub/mod.go

```diff
@@ -5,11 +5,56 @@
 //
 // A concrete instance of a Hub may be an Organizer or a Witness. A baseHub
 // type contains the implementation common across both types.
-//
-// The package also contains an implmentation of the Socket interface which
-// is responsible for low level communication over websockets, i.e. sending
-// and receiving marshaled messages over the wire.
-//
-// The Socket interface has multiple concrete implmentations - one for each
-// type of client the backend interacts with: an Attendee, Organizer or Witness.
 package hub
```

Comment:
Yes!
For future refactoring it would be nice to have the implementation of the hubs in their own package:

```
├─hub
|  ├─base
|  ├─organizer
|  └─witness 
```